### PR TITLE
inconsolata: 1.010 -> inherit google-fonts

### PR DESCRIPTION
**Pull request by mikoim**
###### Motivation for this change
The upstream moved to Google fonts repository on GitHub.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] ~Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))~
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] ~Tested execution of all binary files (usually in `./result/bin/`)~
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/fonts/inconsolata/default.nix [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
   name = "inconsolata-${version}";
+  version = "2.001";
 
-  url = "http://www.levien.com/type/myfonts/Inconsolata.otf";
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "fonts";
+    rev = "4c3e95c802f8f12b78869ff50d552014de63f9c1";
+    sha256 = "1ndmsf4c0k36dakmps0vr7hhg5ss8m7ywja7v55xdrinvli58v2f";
+  };
```
Comment (Mic92) [inline pkgs/data/fonts/inconsolata/default.nix]: We have a package called google-fonts. Maybe we can take the source from there?

```
inherit (google) src version;
```

Comment (mikoim) [reply pkgs/data/fonts/inconsolata/default.nix]: Oh, I missed it. I think that it's nice to have the individual package regarding backward compatibility and package size. (`google-fonts` is bigger than the individual package.) If it's complicated to maintain an individual package, there is another method that split `google-fonts` into multiple packages (e.g., google-fonts: install all fonts, google-fonts-foobar: install foobar font only).

To be honest, I don't know whatever method is optimal.

**Comment by mikoim** [comment]
For the time being, inherit source and version from google-fonts.

**Comment by jb55** [comment]
I'm trying to find out why my fonts look garbage after my latest update to the unstable channel.... perhaps it was this. I will try to revert and report back.

**Comment by Mic92** [comment]
volth wrote:
> Why this is removed?
> It is here for #27754

@volth if somebody updates `google-fonts` this derivation might break. Updating derivations should not have unexpected side effects in other places.

**Comment by mikoim** [comment]
@jb55 Sorry for the inconvenience. What issue happened in your environment?

**Pull request merged**